Show a message when loading video and annotation files

matthewma7 opened the pull request:
This PR will make it show a message when loading video and annotation files using the same location and style as the existing no-annotation message

matthewma7 committed:
Show a message when loading video and annotation files

mwoehlke-kitware commented on web_external/Viewer/index.jsx:
```diff
@@ -63,9 +64,8 @@ class Viewer extends PureComponent {
-                        && <div className='no-annotation-message' key='no-annotation-message'>
-                            <span>No annotation</span>
+                        message && <div className='message' key='message'>
+                            <span>{message}</span>
```
BTW, I never asked, but what is the purpose of the `<span>`?

mwoehlke-kitware commented on web_external/Viewer/index.jsx:
```diff
@@ -63,9 +64,8 @@ class Viewer extends PureComponent {
-                        this.state.ready && !this.props.annotationGeometryContainer
-                        && <div className='no-annotation-message' key='no-annotation-message'>
-                            <span>No annotation</span>
+                        message && <div className='message' key='message'>
```
Do we really want the 'loading' message to be red also, or should we have different classes for different types of message? (If you like, I could do that in a follow-up PR...)

mwoehlke-kitware reviewed (commented):
Some comments / side observations, but I'd be happy merging it as-is and maybe tweaking it further in a new PR.

(Not approving only because I can't test right now due to presumably-unrelated issues.)

mwoehlke-kitware committed:
Tweak 'loading' watermark

Tweak internal message helper to also return a list of CSS classes.
Tweak CSS for watermark messages to use a neutral color for the neutral
'loading' message.
